projectroles/tests/test_templatetags.py

```diff
@@ -618,3 +618,75 @@ def test_get_sidebar_app_legend(self):
         self.assertEqual(
             tags.get_sidebar_app_legend('Update Project'), 'Update<br />Project'
         )
+
+    def test_get_sidebar_links_homepage(self):
+        """Test get_sidebar_links() on the homepage"""
+        url = reverse('home')
+        req_factory = RequestFactory()
+        with self.login(self.user):
+            request = req_factory.get(url)
+            request.resolver_match = resolve(url)
+            request.user = self.user
+            self.assertEqual(
+                tags.get_sidebar_links(request, self.project),
```

Comment:
This expected result is in fact wrong. In the home view, we get no sidebar links, except for superusers who get the "create category" link. I've tested the implementation and it works correctly, so this test must be malfunctioning.